src/unstract/adapters/x2text/llama_parse/src/static/json_schema.json

```
@@ -0,0 +1,46 @@
+{
+    "title": "Llama Parse X2Text",
+    "type": "object",
+    "required": [
+      "api_key"
+    ],
+    "properties": {
+      "adapter_name": {
+        "type": "string",
+        "title": "Llama Parse Adapter ID",
```

```suggestion
        "title": "Name",
```
PR #37 makes this change